cli/cmd/service/remove.go

```diff
@@ -0,0 +1,57 @@
+package service
+
+import (
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func RemoveCmd(ch *cmdutil.Helper) *cobra.Command {
+	var service, projectName string
+
+	removeCmd := &cobra.Command{
+		Use:   "remove",
```

Comment:
Not taking service name here as arg to keep it same as user remove cmd however it feels more consistent to take name as arg in the command

Reply:
please review again